include/arch/arm/aarch64/arm_mmu.h

```diff
@@ -67,6 +67,12 @@
 #define MT_P_RX_U_RX		(MT_RO | MT_RW_AP_ELx | MT_P_EXECUTE | MT_U_EXECUTE)
 #define MT_P_RX_U_NA		(MT_RO | MT_RW_AP_EL_HIGHER  | MT_P_EXECUTE | MT_U_EXECUTE_NEVER)
 
+#if IS_ENABLED(CONFIG_ARMV8_A_NS)
+#define MT_SECURE_ATTR MT_NS
+#else
+#define MT_SECURE_ATTR MT_SECURE
+#endif
```

Comment:
`MT_SECURE_ATTR` is a bit too opaque to me, what about `MT_DEFAULT_SECURE_STATE` or something like that to point out that we are using the "default" state Zephyr was compiled for?

Also, please use `ifdef` for consistency.

Reply:
fix in next version.